sdk/typescript/src/types/objects.ts

```diff
@@ -218,6 +218,12 @@ export function shouldUseOldSharedObjectAPI(version: string): boolean {
   return version === '0.11.0' || version === '0.12.0';
 }
 
+export function shouldUseIntentSigning(version: string): boolean {
```

Comment:
We should use the parsed API version that @666lcz added.

Reply:
This is the function that Jordan refers to: https://github.com/MystenLabs/sui/blob/e45b188a80a067700efdc5a099745f18e1f41aac/sdk/typescript/src/types/version.ts#L10